.bonsai.yml

```diff
@@ -41,14 +41,6 @@
     -  "entity.system.os == 'darwin'"
     -  "entity.system.arch == 'amd64'"
   
-  - platform: "OSX"
-    arch: "386"
-    asset_filename: "#{repo}_#{version}_darwin_386.tar.gz"
-    sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
-    filter:
-    -  "entity.system.os == 'darwin'"
-    -  "entity.system.arch == '386'"
-  
```

Comment:
See [my comment on `sensuctl-plugin-template`](https://github.com/sensu-community/sensuctl-plugin-template/pull/2/files#r377245026)

Reply:
https://github.com/sensu-community/handler-plugin-template/pull/4#discussion_r377097932

Reply:
I've been convinced by @palourde that we can drop that, even for checks.